Gemfile

```diff
@@ -77,4 +75,6 @@ group :test do
   gem 'vcr'
   gem 'webmock'
   gem 'shoulda-matchers'
+  gem 'database_cleaner'
```

Comment:
Do we need `database_cleaner` after all?

We are not using it in `rails_helper` and we have the rspec setting `use_transactional_fixtures = true`, rspec docs say to have that setting to true when not using database cleaner. https://relishapp.com/rspec/rspec-rails/docs/transactions